app/views/guests/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Guest Name: <%= @guest.name %></h1>
+<% @guest.rooms.each do |room| %>
+  <p>Suite: <%= room.suite %></p>
+  <p>Nightly Rate: <%= room.rate %></p>
+  <p>Hotel: <%= room.hotel.name %></p>
+<% end %>
+<div class="new_room">
+  <%= form_with url: "/guest_rooms/#{@guest.id}", method: :post, local: true do |f| %>
```

Comment:
To get the guest id passed in the request, you could add it to the path but a more RESTful route would be `"/guests/guest_id/rooms"` to indicate that the id in the path is for the guest, not the guest_rooms. Another option is using [hidden](https://stackoverflow.com/questions/64533667/how-to-add-a-hidden-field-to-form-with) fields in erb forms